### PR TITLE
More dark mode improvements

### PR DESCRIPTION
**Pull request by 33mhz**

### CODE REVIEW

File: assets/css/main.scss [hunk 1]
```diff
@@ -52,148 +52,167 @@ body.dark .jumbotron {
...
+  }
+
+  .dropdown-item {
+    color: #d5d5d5;
+  }
+  .dropdown-item:hover, .dropdown-item:focus {
+    background-color: #1a1a1a;
+  }
+
+  .list-group-item[data-v-1084c69e]:hover:not(.active):not(.h5) {
```
Comment (sunya9) [inline assets/css/main.scss]: Umm... I don't understand this class that which components. Can you write as scoped css in a vue component?

**Comment by 33mhz** [comment]
‎Not sure what you mean; it's just css. I can't quite remember what this rule is fixing, though. Something quite specific.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        From: sunyaSent: Sunday, March 18, 2018 8:18 AMTo: sunya9/betaReply To: sunya9/betaCc: 33MHz; AuthorSubject: Re: [sunya9/beta] More dark mode improvements (#99)@sunya9 commented on this pull request.



In assets/css/main.scss:
> +    background-color: #252525;
+  }
+
+  .dropdown-menu {
+    background-color: #252525;
+    color: #d5d5d5;
+  }
+
+  .dropdown-item {
+    color: #d5d5d5;
+  }
+  .dropdown-item:hover, .dropdown-item:focus {
+    background-color: #1a1a1a;
+  }
+
+  .list-group-item[data-v-1084c69e]:hover:not(.active):not(.h5) {

Umm... I don't understand this class that which components. Can you write as scoped css in a vue component?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/sunya9/beta","title":"sunya9/beta","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/sunya9/beta"}},"updates":{"snippets":[{"icon":"PERSON","message":"@sunya9 commented on #99"}],"action":{"name":"View Pull Request","url":"https://github.com/sunya9/beta/pull/99#pullrequestreview-104797022"}}}

File: assets/css/main.scss (hunk 1 above)
Comment (33mhz) [reply assets/css/main.scss]: Removed offending code. I think it wasn't necessary anymore.

**Comment by 33mhz** [comment]
Or did you mean something else?

**Pull request merged**